test/functional/signature/version_sig_check.bats

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME"
+	create_bundle -L -n test-bundle1 -f /file_1 "$TEST_NAME"
+       	create_version -p "$TEST_NAME" 20 10
+        update_bundle "$TEST_NAME" test-bundle1 --add file_3
+
+}
+
+@test "SIG019: Verify signature for latest_version" {
+	
+	# Verify the latest_version signature though openssl
+	run sudo sh -c "openssl smime -verify -in $WEBDIR/version/latest_version.sig -inform DER -content $WEBDIR/version/latest_version -purpose any -CAfile $FUNC_DIR/Swupd_Root.pem"
+	expected_output=$(cat <<-EOM
+				Verification successful
+				20
+			EOM
+			)
+
+	assert_is_output "$expected_output"
+
+	# Update succeeds as verification is successful
+	run sudo sh -c "$SWUPD update $SWUPD_OPTS"
+	assert_status_is "$SWUPD_OK"
+
+	expected_output=$(cat <<-EOM
+		Update started
+		Preparing to update from 10 to 20
+		Downloading packs for:
+		 - test-bundle1
+		Finishing packs extraction...
+		Statistics for going from version 10 to version 20:
+		    changed bundles   : 1
+		    new bundles       : 0
+		    deleted bundles   : 0
+		    changed files     : 0
+		    new files         : 1
+		    deleted files     : 0
+		No extra files need to be downloaded
+		Staging file content
+		Applying update
+		Update was applied
+		Calling post-update helper scripts
+		Update successful - System updated from version 10 to version 20
+	EOM
+	)
+	assert_is_output "$expected_output"
+
+}
+
+@test "SIG020: Verify fail signature for latest_version " {
+
+	write_to_protected_file "$WEBDIR/version/formatstaging/latest.sig" "bad signature"
+
+        # Update fails as signature match error occurs as expected
+	run sudo sh -c "$SWUPD update $SWUPD_OPTS"
+        assert_status_is "$SWUPD_SIGNATURE_VERIFICATION_FAILED"
+
+        expected_output=$(cat <<-EOM
+	Error: Unable to determine the server version as signature verification failed
```

Comment:
please add a TAB to each one of these lines of expected output to match the format of other test cases

Reply:
Done